src/System.Windows.Forms/src/System/Windows/Forms/TabPage.cs

```diff
@@ -459,6 +484,66 @@ protected override void OnEnter(EventArgs e)
                 }
 
                 _enterFired = false;
+                KeyboardToolTipStateMachine.Instance.NotifyAboutGotFocus(this);
+            }
+        }
+
+        internal override Rectangle GetToolNativeScreenRectangle()
+        {
+            if (ParentInternal is TabControl tabControl)
+            {
+                return GetItemRectangle(tabControl.SelectedIndex);
+            }
+
+            return Rectangle.Empty;
+        }
+
+        private Rectangle GetItemRectangle(int index)
+        {
+            if (index < 0)
+            {
+                return Rectangle.Empty;
+            }
+
+            RECT rectangle = new RECT();
+            UnsafeNativeMethods.SendMessage(new HandleRef(Parent, Parent.Handle), (int)ComCtl32.TCM.GETITEMRECT, index, ref rectangle);
```

Comment:
Please use `SendMessageW` method instead. 
It would look something like this:
```suggestion
            User32.SendMessageW(Parent, (WindowsMessage)ComCtl32.TCM.GETITEMRECT, (IntPtr)index, ref rectangle);
```

Reply:
Thanks, fixed.